idea/testData/debugger/fileRanking/lambdas.kt

```diff
@@ -33,4 +33,6 @@ class A {
             block { block { val x = 4 }}
         }
     }
-}
\ No newline at end of file
+}
+
+// IGNORE_BACKEND: JVM_IR
```

Comment:
I usually see this in the top of the test files. Is there any consideration putting it here?

Reply:
There will be line number check for some of the test cases, although such is not present in this test case, just make it a convention to reduce future impact on line numbers.